rust/main/submitter/src/chain_tx_adapter/chains/sealevel/adapter.rs

```diff
@@ -267,6 +270,9 @@ impl AdaptsChain for SealevelTxAdapter {
     }
 
     async fn submit(&self, tx: &mut Transaction) -> Result<(), SubmitterError> {
+        if tx.hash.is_some() {
+            return Ok(());
+        }
```

Comment:
this is the most consequential change, see the PR description as well:
> In the SVM adapter, if submit is called on a tx that already has a hash, the submission is skipped. This is because in solana we shouldn't resubmit unless the hash was dropped, and that would be detected when status is queried within the inclusion stage, causing the tx to be dropped, payloads sent back to the building queue, and a new tx built. This can be optimized but there was no obvious simple way to fix for now